.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * 0' # @weekly
+
+jobs:
+  test:
+    name: Python ${{ matrix.python-version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
+        # Exclude Python 3.5 on windows (compiler missing)
+        exclude:
+          - os: windows-latest
+            python-version: 3.5
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install pip and tox
+      run: python -m pip install --upgrade pip tox
+    - name: Install vcpython27
+      if: matrix.os == 'windows-latest' && matrix.python-version == '2.7'
+      run: choco install vcpython27
```

Comment:
choco, love it

Reply:
Won't be needed for long though (hopefully). #370 will make it obsolete.